Makefile

```diff
@@ -26,16 +26,20 @@ bls-zexe: vendor/github.com/celo-org/bls-zexe/bls/target/release/libbls_zexe.a
 
 check_android_env:
 	@test $${ANDROID_NDK_HOME?Please set environment variable ANDROID_NDK_HOME}
-	@test $${NDK_VERSION?Please set environment variable NDK_VERSION}
 
 ndk_bundle: check_android_env
+	@test $${NDK_VERSION?Please set environment variable NDK_VERSION}
 	curl --silent --show-error --location --fail --retry 3 --output /tmp/$(NDK_VERSION).zip \
 		https://dl.google.com/android/repository/$(NDK_VERSION)-linux-x86_64.zip && \
 	mkdir $(ANDROID_NDK_HOME) && \
 	unzip -q /tmp/$(NDK_VERSION).zip -d $(ANDROID_NDK_HOME) && \
 	rm /tmp/$(NDK_VERSION).zip
 
 bls-zexe-android: check_android_env
+	rustup target add aarch64-linux-android
```

Comment:
why not add a check to ensure that rustup is defined command?

Reply:
Right. Let me do that.

Reply:
Done, take a look?